lib/ansible/modules/cloud/amazon/ec2_vpc_net_facts.py

```diff
@@ -193,12 +193,18 @@ def describe_vpcs(connection, module):
 
     # We can get these results in bulk but still needs two separate calls to the API
     try:
-        cl_enabled = connection.describe_vpc_classic_link(VpcIds=vpc_list)
+        if connection._client_config.region_name != 'us-east-1':
```

Comment:
just catch the unsupported operation exception and ignore it. No need to do region checking.